mscore/musescore.cpp

```diff
@@ -1190,7 +1196,7 @@ MuseScore::MuseScore()
     QActionGroup* ag = Shortcut::getActionGroupForWidget(MsWidget::MAIN_WINDOW);
     ag->setParent(this);
     addActions(ag->actions());
-    connect(ag, SIGNAL(triggered(QAction*)), SLOT(cmd(QAction*)));
+    connect(ag, SIGNAL(triggered(QAction*)), SLOT(cmd(QAction*))); // RIGHT HERE
```

Comment:
`// RIGHT HERE`?

Reply:
Hm... oh right, leftover from when I was working on getting playback to work. Removed.